tests/integration/testsuites/custom-domain/manifests/resources.yaml

```diff
@@ -11,7 +22,6 @@ spec:
   type: google-clouddns
   secretRef:
     name: google-credentials
-    namespace: default
```

Comment:
Should we drop the namespace field here? In the Secret defined above, there is still a {{.Namespace}} . Is the default for this field the namespace of the DNSProvider resource?

Reply:
Yes, so the secret must have a namespace field, so it is created in this namespace. Previously it was always created in the 'default' namespace.
In the DNSProvider we now reference the google-credentials secret *in the same namespace*, so we don't need to specify it. Previously the secret was in the different namespace ('default' one), so it was a cross-namespace reference, thus namespace had to be provided.